docker-compose.yml

```diff
@@ -16,15 +16,42 @@ services:
       MSSQL_DB_USER: sa
       MSSQL_DB_PASSWORD: "yourStrong(!)Password"
       MSSQL_DB_DATABASE: test
+      MSSQL_DB_SSH_KEY_PRIVATE: |-
```

Comment:
I got a bad feeling about this...

Reply:
To je schvalne, nemozem to dat do `.env` pretoze to nepodporuje multilne stringy poriadne :/

Reply:
@pivnicek nice Marc, thanks. Mohli by sa aj generovat.